src/client/api/events/node_event.h

```diff
@@ -113,6 +113,14 @@ int event_unsubscribe(event_client_handle_t client, int *mid, char *topic);
  */
 int event_start(event_client_handle_t client);
 
+/**
+ * @brief Mqtt disconnect
```

Comment:
please give more info about what this method does, like
```suggestion
 * @brief Disconnect from the MQTT broker
```

Reply:
Done.